.github/workflows/firefox-ios-ui-tests.yml

```diff
@@ -207,11 +207,12 @@ jobs:
     run-fullfunctionaltests:
       name: Run full functional tests
       runs-on: macos-14
-      needs: compile
+      needs: run-smoketests
+      if: false
       strategy:
         fail-fast: false
         matrix:
-          ios_simulator: ['iPhone 15 Pro Max']
+          ios_simulator: ['iPhone 15 Pro Max', 'iPad Pro (12.9 inch) (6th generation)']
```

Comment:
Running all tests in both platforms exceeds the 4-hour timelimit imposed by Github Actions. But let me put this in just in case we enable this job again.